api/metrics/annually.py

```diff
@@ -0,0 +1,29 @@
+from api.utils.database import rows_to_dicts
+
+
+class AnnuallyMetrics:
+    """
+    Metrics for annually timeline view.
+    """
+
+    def __init__(self, con):
+        self.con = con
+
+    def belonging(self, segment):
+        """
+        Returns the rate of belonging of a segment per year
+        """
+        query = """
+        SELECT
+            period_end_year || "-01-01" as week,
```

Comment:
This is correct, but for some reason, the frontend renders the date as December 31st. I think it's a timezone issue on the frontend, I will look into that and fix.